compiler/control/OMROptions.cpp

```diff
@@ -4643,13 +4688,15 @@ OMR::Options::setBitsFromStringSet(char *option, void *base, TR::OptionTable *en
          }
       }
 
+   TR_VerboseLog::vlogRelease();
    return option;
    }
 
 
 char *OMR::Options::clearBitsFromStringSet(char *option, void *base, TR::OptionTable *entry)
    {
    int i;
+   TR_VerboseLog::vlogAcquire();
```

Comment:
Another place where we are acquiring the lock with the possibility of never actually doing a write.